index.html

```diff
@@ -72,194 +72,146 @@ <h2 class="connect" href="#">LET'S CONNECT</h2>
       </div>
     </section>
 
-    <section class="work-container" id="containers">
-      <div class="container1" id="container1" data-modal-target="#modal1">
-        <img
-          class="desktop-image"
-          src="Images/Desktop-img1.png"
-          alt="Project Image"
-        />
-        <img
-          class="snapshoot"
-          src="Images/Snapshoot1.png"
-          alt="Project Image"
-        />
-        <div class="daily">
-          <h3 class="Tonic">Tonic</h3>
-          <ul class="offer">
-            <li class="canopy">canopy</li>
-            <span class="point"></span>
-            <li class="backend">Back End Dev</li>
-            <span class="point"></span>
-            <li class="year">2015</li>
-          </ul>
-          <p class="description">
-            A daily selection of privately personalized reads; no accounts or
-            sign-ups required.
-          </p>
-
-          <ul class="skills">
-            <li class="htl"><a href="#"></a>html</li>
-            <li class="css"><a href="#"></a>css</li>
-            <li class="js"><a href="#"></a>javascript</li>
-          </ul>
-          <button class="seeproject" data-modal-target="#modal1">See Project</button>
-        </div>
-      </div>
-
-      <div class="container2 desktop-reverse" id="container2" data-modal-target="#modal0">
-        <img
-          class="desktop-image"
-          src="Images/Desktop-img2.png"
-          alt="Project Image"
-        />
-        <img class="snapshoot" src="Images/Snapshoot2.png" alt="mobile image" />
-        <div class="daily">
-          <h2 class="Tonic2">Multi-Post Stories</h2>
-          <ul class="offer">
+    <section class="work-container" id="container">
+      <div class="container1" id="portfolio">    
+      <img class="desktop-image" src="Images/Desktop-img1.png" alt="Desktop Image">
+      <img class="snapshoot" src="Images/Snapshoot1.png" alt="mobile image">
+      <div class="daily">
+        <h3 class="Tonic">Tonic</h3>
+        <ul class="offer">
+           <li class="canopy">canopy</li>
+           <span class="point"></span>
+           <li class="backend">Back End Dev</li>
+           <span class="point"></span>
+           <li class="year">2015</li>
+        </ul>
+         <p class="description">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
```

Comment:
- [ ] I also noticed that the mobile menu only opens once. Once you open and close the mobile menu, It won't open again. Please look into this :smile: :+1: 
- VIDEO FOR REFERNCE:

https://user-images.githubusercontent.com/107632780/232638186-f4b8699c-2abb-4de8-9ff5-9dcea701324c.mp4